corehq/apps/reports/templatetags/xform_tags.py

```diff
@@ -35,7 +35,7 @@
 
 @register.simple_tag
 def render_form_xml(form):
-    xml = form.get_xml()
+    xml = form.get_xml().encode('utf-8')
```

Comment:
@millerdev can advise but i think it might be better to check if it's unicode first(?)